src/screens/Help/HelpScreen.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import { View, Text, Image, TouchableOpacity, Modal, Alert, Pressable, StyleSheet, Dimensions } from 'react-native'
+import { RFPercentage } from 'react-native-responsive-fontsize';
+import { Ionicons } from '@expo/vector-icons';
+
+//components
+import Screen from './../components/Screen';
+//config
+import heme from '../consts/theme';
+import { LinearGradient } from 'expo-linear-gradient';
+import { COLORS } from '../../consts/theme';
+
+
+const { height } = Dimensions.get("window");
```

Comment:
You can get the height of the screen theme.js file
```suggestion
import { SIZES } from '../../consts/theme';

style={{ height: SIZES.height }}
```

Reply:
I tried using that but somehow there are a few conflicts that I fear might happen with that. I would definitely try this moving forward. For now, I have tried using that but just defined the const as SIZES.height above and imported the SIZES from the theme. Would love to see the code implementation though. Thank you for the input, I also corrected a few issues with paths that I put wrong somehow my IDE didn't save changes so pushing that now. The screen seems to work fine!  
![image](https://user-images.githubusercontent.com/11522563/139574553-6293f65f-d531-4a00-99eb-5a4870eca227.png)

Reply:
@Kapersky1337 What specific conflicts do you fear might happen? Did you implement it and did they happen?

@sebastiaohns Can you share a code implementation for @Kapersky1337 ?

Reply:
@steffenkrogmann I have understood the point, I misunderstood a point regarding the theme.js implementation where I thought clashes might happen, I mixed two things in one. 

In this scenario, I quite grasped that a stylesheet has to be created with the components that are being used again and again (for eg. text components) and group them together into a const style. 

I would go ahead and implement the following. Would ask @sebastiaohns if any issue persists.